### PR TITLE
Add unit tests

### PR DESCRIPTION
**Pull request by jeremy-then**

### CODE REVIEW

File: samples/transferBtc-samples.js [hunk 1]
```diff
@@ -76,7 +78,9 @@ const printP2SHMULTISIGTransferHash = async () => {
     const transferAmountInBtc = 3;
 
     const btcTransactionHelper = new BtcTransactionHelper(config);
-    const sender = await btcTransactionHelper.generateMultisigAddress(3, 2, 'legacy');
+    const sender = await btcTransactionHelper.generateMultisigAddress(2, 2, 'legacy');
+    //console.log('sender: ', sender)
```
Comment (marcos-iov) [inline samples/transferBtc-samples.js]: Remove commented out code please

File: test/btc-transaction-helper.test.js [hunk 2]
```diff
@@ -0,0 +1,449 @@
...
+    it('should import address', async () => {
+
+        const btcTransactionHelper = new BtcTransactionHelper(config);
+        const nodeClient = btcTransactionHelper.nodeClient;
+        const importAddressStub = sinon.stub(nodeClient, 'execute').resolves(null);
+        const result = await btcTransactionHelper.importAddress(TEST_BTC_ADDRESS, 'label');
+
+        assert.isNull(result);
+        assert.isTrue(importAddressStub.calledOnce);
+        assert.isFalse(importAddressStub.calledWith(TEST_BTC_ADDRESS, 'label'));
```
Comment (marcos-iov) [inline test/btc-transaction-helper.test.js]: Why is this false? isn't that exactly the call that was made?

**Review by marcos-iov — COMMENTED** [review]
Excellent piece of Jere. Congrats!

File: test/btc-transaction-helper.test.js (hunk 2 above)
Comment (jeremy-then) [reply test/btc-transaction-helper.test.js]: I found out that that test is duplicated. This specific test passes because the `execute` is not called with those values like that, instead it is called like `execute('importaddress', [address, label])`, that's why this test passes. I fixed it and removed the duplicate test below.

File: test/btc-transaction-helper.test.js [hunk 3]
```diff
@@ -0,0 +1,449 @@
...
+        amount: 2000,
+    },
+    {
+        amount: 3000,
+    }
+];
+
+describe('BtcTransactionHelper', () => {
+
+    it('should import address', async () => {
```
Comment (kelvinator07) [inline test/btc-transaction-helper.test.js:51]: This Tests Also Exists In Line 346

File: test/btc-transaction-helper.test.js [hunk 4]
```diff
@@ -0,0 +1,449 @@
...
+
+        const btcTransactionHelper = new BtcTransactionHelper(config);
+        const nodeClient = btcTransactionHelper.nodeClient;
+        const executeImportAddressStub = sinon.stub(nodeClient, 'execute').resolves(null);
```
Comment (kelvinator07) [inline test/btc-transaction-helper.test.js:55]: Why Resolve To `null` Instead Of The Actual Address?

File: package.json
Comment (kelvinator07) [inline package.json:1]: You Should Also Increase The Version Since You Have Added Unit Tests

Comment (marcos-iov) [reply package.json:1]: Not really, doesn't change any functionality

File: test/btc-transaction-helper.test.js (hunk 2 above)
Comment (marcos-iov) [reply test/btc-transaction-helper.test.js]: Didn't see any test removed? Did you forget to push changes?

File: test/btc-transaction-helper.test.js (hunk 4 above)
Comment (jeremy-then) [reply test/btc-transaction-helper.test.js:55]: Because `importAddress` returns `null`. Check the type: `importAddress(address: string, label: string): Promise<null>;`

**Review by marcos-iov — APPROVED** [review]

File: samples/transferBtc-samples.js [hunk 5]
```diff
@@ -76,7 +78,9 @@ const printP2SHMULTISIGTransferHash = async () => {
     const transferAmountInBtc = 3;
 
     const btcTransactionHelper = new BtcTransactionHelper(config);
-    const sender = await btcTransactionHelper.generateMultisigAddress(3, 2, 'legacy');
+    const sender = await btcTransactionHelper.generateMultisigAddress(2, 2, 'legacy');
```
Comment (nathanieliov) [inline samples/transferBtc-samples.js:81]: Why did you change the signer size from 3 to 2?

**Review by nathanieliov — APPROVED** [review]
LGTM
